src/clj/puppetlabs/puppetserver/certificate_authority.clj

```diff
@@ -69,7 +69,19 @@
    :ruby-load-path           [schema/Str]
    :gem-path                 schema/Str
    :signeddir                schema/Str
-   :serial                   schema/Str})
+   :serial                   schema/Str
+   ;; This would be list of compile master node names (FQDNs) including MoM
+   ;; It would either be provisioned by PE or created & maintained by user (input configuration)
+   :compile-masters          [schema/Str]    
+   ;; This would point to a file containing a list of serial numbers of compile master certificates
+   ;; corresponding to the nodes from the above file. This would be re-generated everytime the
+   ;; compile-masters list is updated.
+   :compile-master-serials-path   schema/Str
+   ;; Path to Infrastructure CRL file containing master certificates
+   :infra-crl-path                schema/Str
+   ;; Option to continue using full CRL instead of infrastructure CRL if desired
+   ;; Infrastructure CRL would be enabled by default.
+   :disable-infra-crl         schema/Bool })
```

Comment:
I think when we do settings we should keep them positive, eg `enable-infra-crl`. Sometimes negative settings confuse folks.